x-pack/plugins/secops/public/components/drag_and_drop/drag_drop_context_wrapper.tsx

```diff
@@ -63,3 +65,19 @@ const mapStateToProps = (state: State) => {
 };
 
 export const DragDropContextWrapper = connect(mapStateToProps)(DragDropContextWrapperComponent);
+
+const disableScrolling = () => {
+  const x =
+    window.pageXOffset !== undefined
+      ? window.pageXOffset
```

Comment:
Looks like we're [good](https://stackoverflow.com/a/46962472) on IE11 with this too -- so double thanks on this fix! :)